snekdancer/settings.py

```diff
@@ -115,9 +119,27 @@
 
 USE_TZ = True
 
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-# Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/2.2/howto/static-files/
+# this line is already in your settings.py
```

Comment:
Is this the right comment? Should it refer more to something else?

Reply:
Lol oops that is copied from a walkthrough